### PR TITLE
Dockerization of Apache fuseki done

### PR DESCRIPTION
**Pull request by maqboolkhan**
- [x] Added Apache Jena image to our stack
- [x] Added DB to Docker network
- [x] Data persistence even after removing the stack and starting it again

### CODE REVIEW

**Comment by maqboolkhan** [comment]
https://github.com/dice-group/ida/issues/134

File: services/docker-compose-dev.yml [hunk 1]
```diff
@@ -20,6 +20,18 @@ services:
       - "../ida-ws/target:/app/ida-ws/target"
     networks:
       - ida-net
+  fuseki:
+    image: stain/jena-fuseki
+    ports:
+      - 3030:3030
+    environment:
+      - ADMIN_PASSWORD=pw123
```
Comment (nikit91) [inline services/docker-compose-dev.yml]: password in plain text

Comment (Cortys) [reply services/docker-compose-dev.yml]: Should be fixed now. The password can now be configured via the `FUSEKI_PW` file.

I still included a default password in the updated stack to make the development setup a bit easier. For a production deployment, the password can be reconfigured easily.

Comment (maqboolkhan) [reply services/docker-compose-dev.yml]: Thanks @Cortys. Understandable!
@nikit91 can you also review and merge?

**Pull request merged**